ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/hooks/useAffectedImagesCount.ts

```diff
@@ -0,0 +1,49 @@
+import { gql, useQuery } from '@apollo/client';
+import { getImageScope } from '../utils';
+import { getRequestQueryStringForSearchFilter } from 'utils/searchUtils';
+import { VulnerabilityException } from 'services/VulnerabilityExceptionService';
+
+type AffectedImagesCountQueryResult = {
+    imageCVEs: {
+        cve: string;
+        affectedImageCount: number;
+    }[];
+};
+
+type UseAffectedImagesCountResult = {
+    isAffectedImagesCountLoading: boolean;
+    affectedImagesCount: number;
+};
+
+export const affectedImagesCountQuery = gql`
+    query getAffectedImagesCount($query: String) {
+        imageCVEs(query: $query) {
+            cve
```

Comment:
Do we need to query the `cve` field here?

Reply:
Actually now that I think of it, I believe we are going to need to reuse this hook later to show the affected image count of individual CVEs in the "update" modal.

What do you think of the following in a later PR?
- Rename `affectedImagesCount` to `totalAffectedImageCount`
- Return the raw `data` as well, so we can get individual counts by CVE